documentation/src/main/asciidoc/mapper-orm-configuration.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[mapper-orm-configuration]]
+= General configuration
+// Search 5 anchors backward compatibility
+[[search-configuration-event]]
+
+== Enabling the integration
+
+The Hibernate ORM integration is enabled by default as soon as it is present in the classpath.
+
+If for some reason you need to disable it,
+set the `hibernate.search.autoregister_listeners` <<configuration-property-types,boolean property>> to `false`.
+
+== Configuration property tracking
```

Comment:
How is this paragraph related to ORM?

Reply:
The feature is implemented in the ORM integration, because in order to detect unused properties, we need a list of all properties, which engine SPIs do not require (we only need it for this particular feature). Other integrations may not be able to provide this feature, for example if properties are retrieved from a datasource that only provides find-by-key retrieval.

Anyway... Ok, I'll admit I don't have a very good point. I'll move this to the top-level "Configuration" section.